influxql/token.go

```diff
@@ -54,8 +55,8 @@ const (
 	SEMICOLON // ;
 	DOT       // .
 
-	keyword_beg
-	// Keywords
+	keywordBeg
+	// ALL and other Keywords
```

Comment:
I agree, but golint wants there to be at least a comment in both places. What would you recommend? I've made these other changes.